src/PieChart/components/__tests__/configs.spec.ts

```diff
@@ -0,0 +1,149 @@
+import * as PieChartConfigs from "../../utils/configs";
```

Comment:
All tests should be in the root PieChart folder... please move the __tests__ folder.